warehouse/templates/email/new-email-added/body.txt

```diff
@@ -14,7 +14,7 @@
 {% extends "email/_base/body.txt" %}
 
 {% block content %}
-{% trans username=username, email_address=email_address %}Hello, {{ username }}. A new email '{{ email_address }}' has been added to your account.{% endtrans %}
+{% trans username=username, new_email_address=new_email_address %}Hello, {{ username }}. A new email '{{ new_email_address }}' has been added to your account.{% endtrans %}
 
-<p>{% trans href='mailto:admin@pypi.org', email_address='admin@pypi.org' %}If you did not make this change, you can email <a href="{{ href }}">{{ email_address }}</a> to communicate with the PyPI administrators.{% endtrans %}</p>
+<p>{% trans href='mailto:admin@pypi.org', new_email_address='admin@pypi.org' %}If you did not make this change, you can email <a href="{{ href }}">{{ new_email_address }}</a> to communicate with the PyPI administrators.{% endtrans %}</p>
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**thought:** ‏I don't think this change was necessary, since it's not a new_email_address, it's the admin one. Guessing Find & Replace got it. Shouldn't be harmful, just a tad inaccurate in the variable naming.
Same comment applies for `body.html`